client/src/css/Alternative.css

```diff
@@ -1,43 +1,45 @@
 .Alternative {
-  margin: 1rem 0;
-  background: #6b71a5;
-  color: #ffffff;
-  font-size: 1.8rem;
+  margin: 0.5rem 0;
+  background-color: #6B71A5;
+  color: #FFF;
   display: flex;
-  box-shadow: 0px -2px 0px 0px rgba(0, 0, 0, 0.25) inset,
-              0px 2px 1px 0px rgba(0, 0, 0, 0.2);
+  box-shadow: 0 -0.125rem 0 0 rgba(0, 0, 0, 0.25) inset,
```

Comment:
I don't have any sources as to why this is "better", but from what I've noticed the browser will round border-width to nearest pixel anyway. 

"Everyone" knows what a 1px thick border looks like, but fiddling with tiny numbers like 0.0625 is just annoying. Sure you could round it, but what do you do when you want the border to be just a bit thicker? Then you have to know (or convert from pixels) to find out the nearest rem that will provide you with that. Overall it's just too much hassle imo.